plugins/techdocs/src/Router.tsx

```diff
@@ -61,17 +61,25 @@ export const EmbeddedDocsRouter = (props: PropsWithChildren<{}>) => {
   const { children } = props;
   const { entity } = useEntity();
 
+  // Using objects instead of <Route> elements, otherwise "outlet" will be null on sub-pages and add-ons won't render
+  const element = useRoutes([
```

Comment:
Did this need to be moved up? Thinking it could still be below the early `!projectId` return?

Reply:
Yes, because we can't call hook conditionally...

![image](https://user-images.githubusercontent.com/6290749/169493924-e00642c8-42ef-4fe9-a614-e8b9e2b7144b.png)